desktop/electron-builder.yml

```diff
@@ -17,7 +18,7 @@ extraResources:
       - "**/*"
   - "ThirdPartyNotices.txt"
   - "data"
-  - "i18n"
+  - "i18n-deprecated"
```

Comment:
Is this correct? I thought @skapur12 got it working with the new translation strings in the `i18n` directory.

Reply:
The change to the desktop app is still on the dpwatrous/fake-localizer feature branch (not merged to main yet), I'll just include it in the next PR with all of the other changes!